### DISCUSSION TITLE
Imported skins should be duplicated as modified on first modification (i.e. protect imported skins)

### DISCUSSION THREAD

**Discussion by ZeroAurora** [ppy/osu#29609] [Ideas] [answered]
I just accidentally cleared out all skin elements of an imported skin and pressed esc, then the skin automatically saved... Without a *modified* version as modified from default skins. I have to redownload that skin.
Should we protect them by duplicating them, just like what we're doing with default skins?

**Comment by bdach** [comment]
How did you *accidentally* clear a skin? That's the part that bothers me most in this description.

**Reply by ZeroAurora** [reply]
[This skin](https://osu.ppy.sh/community/forums/topics/1923469) specifically, after clicking **Revert to default**. Perhaps it's because of the non-standard skinning method it uses?

**Reply by bdach** [reply]
"Revert to default" can be a bit broken (https://github.com/ppy/osu/issues/22562), but it shouldn't be deleting all elements of a skin. It doesn't do it for me when I tried it.

**Reply by ZeroAurora** [reply]
I would suspect that it's the skin's issue. I can reproduce the issue on this skin while I cannot reproduce on others

**Comment by peppy** [comment, marked as answer]
I don't know about this one. Protected skins are only a thing for defaults.

I'd suggest you just redownload it.
